packages/ui/src/Layouts/PageLayoutHeader.tsx

```diff
@@ -91,8 +91,19 @@ const HeaderLink = styled(NavLink)`
     border-image-slice: 1;
   }
 `
+
 const HeaderLinksWrapper = styled.div`
   align-items: center;
   justify-content: center;
   display: flex;
 `
+
+const ChildrenLinksWrapper = styled.div`
+  align-items: center;
```

Comment:
why is ChildrenLinksWrapper needed?

Reply:
let me know please what was missing from PageLayoutHeader component, maybe @MilosMarkoni had some other approach in mind when he developed this.

Reply:
I have it resolved in my branch https://github.com/BloxiFi/bloxifi-fe/pull/156/files, not merged yet though